languages/python/oso/polar/polar.py

```diff
@@ -235,6 +236,9 @@ def register_class(self, cls, *, name=None):
         """Register `cls` as a class accessible by Polar."""
         cls_name = self.host.cache_class(cls, name)
         self.register_constant(cls, cls_name)
+        self.ffi_polar.register_mro(
+            cls_name, self.host.to_polar(list(inspect.getmro(cls)))
```

Comment:
Does that matter thought? All classes have to be registered by this point in order to use them as specializers.

Reply:
Yeah this is a problem that I didn't think of. I guess we need to add all the MROs after every class has been registered, which is unfortunate.

Reply:
Moved registering MROs with Oso from `register_class()` to the `load()` method. They will be re-registered every time `load()` is called, which I think is necessary in case classes are registered between loads. Not great but I don't think it's a big issue.

Reply:
This would be an issue though if you register a class after load, right?

Reply:
I don't think we should fix it now but another point in favor of improving init process for Oso instance.

Reply:
As long as `load()` is called again after registering more classes, it would be fine. It's already an issue to register classes after `load()` because you'll get unrecognized specializer errors. But yeah the init needs to be generally improved.